frappe/share.py

```diff
@@ -101,6 +101,8 @@ def set_permission(doctype, name, user, permission_to, value=1, everyone=0):
 @frappe.whitelist()
 def get_users(doctype, name):
 	"""Get list of users with which this document is shared"""
+	frappe.has_permission(doctype, throw=True)
```

Comment:
```suggestion
	frappe.has_permission(doctype, doc=name, throw=True)
```
User can also have access to the document, if the document has been shared.

Reply:
`frappe.has_permission` doesn't seem so sure about whether it accepts a string for `doc`. But I guess it'll work for most cases 😅